### ISSUE TITLE
Favorites Tab

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kexplo/imame4all#28]
```
Add a favorite games option.  When toggled on the game(s) selected will show up 
on a new favorites/bookmarks tab.


```

Original issue reported on code.google.com by `nschneid...@gmail.com` on 15 Aug 2011 at 3:37

**Comment by GoogleCodeExporter** [comment]
```
Yes. This is a feature that I would like add. And some kind of game filtering. 
```

Original comment by `seleuco....@gmail.com` on 15 Aug 2011 at 9:23

**Comment by GoogleCodeExporter** [comment]
```
Implemented in v1.10.0
```

Original comment by `squid123...@gmail.com` on 22 Sep 2011 at 12:16
- Changed state: **Done**
- Added labels: **Type-Enhancement**
- Removed labels: **Type-Defect**

**Comment by GoogleCodeExporter** [comment]
Original comment by `squid123...@gmail.com` on 22 Sep 2011 at 1:05
- Changed state: **Fixed**
